steely/plugins/mayo.py

```diff
@@ -0,0 +1,44 @@
+'''gives the previous command m a y o n n a i s e'''
+
+import random
+import re
+import string
+
+__author__ = ('iandioch')
+COMMAND = '.mayo'
```

Comment:
dot in command name 💯

Reply:
creds to me for spotting this thank you mr aaron for stealing the spotlight sir